### PR TITLE
fixedissue #148

### PR DESCRIPTION
**Pull request by abhishek11008**

### CODE REVIEW

File: app/src/main/java/org/merakilearn/EnrollActivity.kt [hunk 1]
```diff
@@ -153,6 +153,19 @@ class EnrollActivity : AppCompatActivity() {
...
+            tvClassLanguage.isVisible = true
+            if(it.toLowerCase().equals("hi")) {
+                tvClassLanguage.text = "HINDI"
+            }else if(it.toLowerCase().equals("en")) {
+                tvClassLanguage.text = "ENGLISH"
+            }else{
+                tvClassLanguage.text ="OTHER"
+            }
+        } ?: kotlin.runCatching {
+            tvClassLanguage.isVisible = false
```
Comment (suhaibroomy) [inline app/src/main/java/org/merakilearn/EnrollActivity.kt:166]: Hardcoding is not such a good idea, Maybe we can fetch this from Android via Locale

File: app/src/main/java/org/merakilearn/datasource/network/model/Classes.kt [hunk 2]
```diff
@@ -50,5 +52,5 @@ data class Classes(
 }
 
 fun Classes.sanitizedType(): String = type.replace("_", " ").capitalizeWords()
-
-fun Classes.timeRange(): String = "${startTime.toTime()} - ${endTime.toTime()}"
\ No newline at end of file
+fun Classes.timeRange(): String = "${startTime.toTime()} - ${endTime.toTime()}"
+fun Classes.dateFormatUI(): String = "${startTime.toDate()}"
```
Comment (suhaibroomy) [inline app/src/main/java/org/merakilearn/datasource/network/model/Classes.kt:56]: Knitpicking on this one but UI is not a very intuitive name, something like formattedDate or displayableDate would make more sense. Feel free to ignore this one

File: app/src/main/res/layout/item_class.xml [hunk 3]
```diff
@@ -54,33 +54,56 @@
...
+
+            <TextView
+                android:id="@+id/tvClassTiming"
+                style="?textAppearanceCaption"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_below="@+id/tvClassDate"
+                android:layout_marginEnd="8dp"
+                android:layout_marginTop="8dp"
+                android:layout_marginBottom="4dp"
```
Comment (suhaibroomy) [inline app/src/main/res/layout/item_class.xml:99]: dimension for spacing should not be hardcoded. `spacing_2x` should be used here

File: app/src/main/res/layout/item_class.xml [hunk 4]
```diff
@@ -20,32 +20,32 @@
 
         <androidx.constraintlayout.widget.ConstraintLayout
             android:layout_width="match_parent"
-            android:layout_height="wrap_content">
+            android:layout_height="match_parent">
```
Comment (suhaibroomy) [inline app/src/main/res/layout/item_class.xml:23]: This should be wrap content as we want the card to hug the content.

**Review by suhaibroomy — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/merakilearn/EnrollActivity.kt (hunk 1 above)
Comment (suhaibroomy) [reply app/src/main/java/org/merakilearn/EnrollActivity.kt:166]: runCatching is also not needed here

Comment (abhishek11008) [reply app/src/main/java/org/merakilearn/EnrollActivity.kt:166]: Done

**Review by suhaibroomy — APPROVED** [review]

**Pull request merged**